1st-gen/packages/menu/src/MenuItem.ts

```diff
@@ -640,7 +697,14 @@ export class MenuItem extends LikeAnchor(
         }
     };
 
-    protected handlePointerenter(): void {
+    protected handlePointerenter(event: PointerEvent): void {
+        this._lastPointerType = event.pointerType;
```

Comment:
`_lastPointerType` can become stale across interactions and it should be scoped to the active pointer. You can use `pointerId`